pkg/engine/resolve/resolve.go

```diff
@@ -955,6 +958,27 @@ func (r *Resolver) resolveFloat(ctx *Context, floatValue *Float, data []byte, fl
 	return nil
 }
 
+func (r *Resolver) resolveBigInt(ctx *Context, bigIntValue *BigInt, data []byte, bigIntBuf *BufPair) error {
+	value, valueType, _, err := jsonparser.Get(data, bigIntValue.Path...)
+	switch {
+	case err != nil, valueType == jsonparser.Null:
+		if !bigIntValue.Nullable {
+			return errNonNullableFieldValueIsNull
+		}
+		r.resolveNull(bigIntBuf.Data)
+		return nil
+	case valueType == jsonparser.Number:
+		bigIntBuf.Data.WriteBytes(value)
+	case valueType == jsonparser.String:
+		bigIntBuf.Data.WriteBytes(value)
```

Comment:
in case of string, you have to quote it when writing to buffer

value from json parser comes unquoted

```
	bigIntBuf.Data.WriteBytes(quote)
	bigIntBuf.Data.WriteBytes(value)
	bigIntBuf.Data.WriteBytes(quote)
```

Reply:
Thanks! I've updated it, PTAL